System/Posix/ByteString/FilePath.hsc

```diff
@@ -125,3 +129,9 @@ throwErrnoPathIfMinus1 = throwErrnoPathIf (== -1)
 --
 throwErrnoPathIfMinus1_ :: (Eq a, Num a) => String -> RawFilePath -> IO a -> IO ()
 throwErrnoPathIfMinus1_  = throwErrnoPathIf_ (== -1)
+
+-- | as 'throwErrnoTwoPathsIfMinus1_', but exceptions include two paths when appropriate.
+--
+throwErrnoTwoPathsIfMinus1_ :: (Eq a, Num a) => String -> RawFilePath -> RawFilePath -> IO a -> IO ()
+throwErrnoTwoPathsIfMinus1_  loc path1 path2 =
+    throwErrnoIfMinus1_ (loc <> " '" <> BC.unpack path1 <> "' to '" <> BC.unpack path2 <> "'")
```

Comment:
This is strictly speaking a breaking change: someone could reasonably catch the error message and analyse it.

Reply:
Well, not per PVP, because the API is defined in terms of types only, not values.

Reply:
Sure, I can remove this patchset, since it's not strictly speaking a bugfix.

Reply:
> technically allows me to replace every function call with undefined.

You're giving me ideas :smiling_imp: